### PR TITLE
Add ruby 3 support.

### PR DESCRIPTION
**Pull request by KapitonovRoman**
Fixes ruby version related issues

### CODE REVIEW

**Review by mbhnyc — APPROVED** [review]
looks right

File: lib/stenotype/frameworks/rails/action_controller.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ module ActionControllerExtension
         # @todo What is really the name here?
         #
         def _record_freshly_event(name)
-          Stenotype::Event.emit!(name, { type: "controller_action" }, { eval_context: { controller: self }})
+          Stenotype::Event.emit!(name, { type: "controller_action" }, **{ eval_context: { controller: self }})
```
Comment (pavel-kremen-freshly) [inline lib/stenotype/frameworks/rails/action_controller.rb:27]: `Stenotype::Event.emit!(name, { type: "controller_action" }, eval_context: { controller: self })`

File: lib/stenotype/frameworks/rails/active_job.rb [hunk 2]
```diff
@@ -37,7 +37,7 @@ def trackable_job!
               Stenotype::Event.emit!(
                 "active_job_#{self.class.name}",
                 { type: "active_job" },
-                { eval_context: { active_job: self }},
+                **{ eval_context: { active_job: self }},
```
Comment (pavel-kremen-freshly) [inline lib/stenotype/frameworks/rails/active_job.rb:40]: eval_context: { active_job: self }

File: spec/stenotype/frameworks/rails/active_job_extension_spec.rb [hunk 3]
```diff
@@ -45,7 +45,7 @@ def name
 
   describe ".trackable_job!", type: :with_frozen_time do
     it "prepends an emit event action before performing the job" do
-      expect { job_instance.perform(arg: :value) }.to change { test_buffer.map { |e| JSON.parse(e) } }.to([ expected_result ])
+      expect { job_instance.perform({ arg: :value }) }.to change { test_buffer.map { |e| JSON.parse(e) } }.to([ expected_result ])
```
Comment (pavel-kremen-freshly) [inline spec/stenotype/frameworks/rails/active_job_extension_spec.rb:48]: `test_buffer.map { JSON.parse(_1) }`

File: lib/stenotype/frameworks/rails/action_controller.rb (hunk 1 above)
Comment (KapitonovRoman) [reply lib/stenotype/frameworks/rails/action_controller.rb:27]: this change is intentional to support ruby 3 kwargs

**Pull request merged**